public_html/wp-content/mu-plugins/blocks/assets/src/speakers/edit.js

```diff
@@ -59,6 +67,8 @@ const speakersSelect = ( select, props ) => {
 	}
 
 	const speakersQuery = pickBy( args, ( value ) => ! isUndefined( value ) );
+	// it sounds like `pick` isn't needed in es6, does that also apply to pickBy?
+	// https://www.sitepoint.com/lodash-features-replace-es6/
```

Comment:
Good question. `pick` and `pickBy` don't do exactly the same thing, but it certainly would be worth exploring whether we could eliminate this and maybe other lodash dependencies.